macosx/FileNameCell.mm

```diff
@@ -87,40 +93,36 @@ - (void)drawWithFrame:(NSRect)cellFrame inView:(NSView*)controlView
             respectFlipped:YES
                      hints:nil];
 
-    NSColor *titleColor, *statusColor;
     FileListNode* node = self.objectValue;
+    AttributesStyle style;
     if (self.backgroundStyle == NSBackgroundStyleEmphasized)
     {
-        titleColor = statusColor = NSColor.whiteColor;
+        style = AttributesStyleEmphasized;
```

Comment:
I like that we've moved from talking about individual colors to talking about intent :+1: